### PR TITLE
Release Hermes v1.7.1

### PR DESCRIPTION
**Pull request by romac**
Not to be merged via the GitHub UI. I will rebase it on top of the `v1.7.x` branch instead.

### CODE REVIEW

File: .changelog/v1.7.1/summary.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+This patch release of Hermes now allows operators to set the clearing interval
+at a different value for each chain, using the new per-chain `clear_interval` setting.
+The global `clear_interval` setting is used as a default value if the per-chain
+setting is not defined.
+
+Additionnaly, operators can now set override the CometBFT compatibility mode to be used
```
Comment (ljoss17) [inline .changelog/v1.7.1/summary.md:6]:
```suggestion
Additionnaly, operators can now override the CometBFT compatibility mode to be used
```

File: .changelog/v1.7.1/summary.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+for a chain by using the new `compat_mode` per-chain setting. The main use case for this,
```
Comment (ljoss17) [inline .changelog/v1.7.1/summary.md:7]:
```suggestion
for a chain by using the new `compat_mode` per-chain setting. The main use case for this
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+## v1.7.1
+
+*November 13th, 2023*
+
+This patch release of Hermes now allows operators to set the clearing interval
+at a different value for each chain, using the new per-chain `clear_interval` setting.
+The global `clear_interval` setting is used as a default value if the per-chain
+setting is not defined.
+
+Additionnaly, operators can now set override the CometBFT compatibility mode to be used
```
Comment (ljoss17) [inline CHANGELOG.md:12]:
```suggestion
Additionnaly, operators can now override the CometBFT compatibility mode to be used
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+for a chain by using the new `compat_mode` per-chain setting. The main use case for this,
```
Comment (ljoss17) [inline CHANGELOG.md:13]:
```suggestion
for a chain by using the new `compat_mode` per-chain setting. The main use case for this
```

**Review by ljoss17 — APPROVED** [review]
Great works! I left a few suggestions for the changelog summary

**Comment by romac** [comment]
Not to be merged via the GitHub UI. I will rebase it on top of the v1.7.x branch instead

**Pull request merged**